docs/source/coredev/release_process.rst

```diff
@@ -8,7 +8,15 @@ This document contains the process that is used to create an IPython release.
 
 Conveniently, the ``release`` script in the ``tools`` directory of the ``IPython``
 repository automates most of the release process. This document serves as a
-handy reminder and checklist for the release manager.
+handy reminder and checklist for the release manager. 
+
+During the release process, you might need the extra following dependencies:
+
+ - ``keyring`` to access your GitHub authentication tokens
+ - ``testpath`` to fake some file system operations
```

Comment:
Yes you are right, just filled that form the note I took was @fperez was releasing 5.0 and testpath was one of the missing packages.